.github/workflows/make-release.yaml

```diff
@@ -0,0 +1,75 @@
+name: Make release
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - CITATION.cff
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Debug dump
+        uses: crazy-max/ghaction-dump-context@v2
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Get previous version file
+        run: git show HEAD~1:CITATION.cff >> CITATION-previous.cff
+
+      - name: Install packages
+        run: npm install yaml@v2 semver@v7 glob@v11
+
+      - name: Get version
+        id: version
+        uses: actions/github-script@v7
+        with:
+          result-encoding: string
+          script: |
+            const { readFileSync, renameSync } = require("fs");
+            const { valid, eq, lt } = require("semver");
+            const { parse } = require("yaml");
+            const { globSync } = require("glob");
+
+            // load and parse file contents
+            const { version: newVersion } = parse(readFileSync("CITATION.cff").toString());
+            const { version: oldVersion } = parse(readFileSync("CITATION-previous.cff").toString());
+
+            console.log(`Old version: ${oldVersion}`);
+            console.log(`New version: ${newVersion}`);
+
+            // check version
+            if (!valid(newVersion) || lt(newVersion, oldVersion))
+              throw Error("Version not valid");
+            if (eq(oldVersion, newVersion)) {
+              console.log("Version unchanged");
+              return "";
+            }
+
+            // add version to artifact filenames
+            for (const file of globSync("**/STRchive-*.json"))
+              renameSync(file, file.replace(".json", `_v${newVersion}.json`));
+            for (const file of globSync("**/*STRchive-disease-loci*.bed"))
+              renameSync(file, file.replace("-loci", `-loci_v${newVersion}`));
```

Comment:
If you're going to keep the file names as they are, I'd change the `_` to a `.` to be consistent with the other separators. As it is right now, it'd become e.g. `T2T-chm13.STRchive-disease-loci_v2.1.0.TRGT.bed`.

Ideally though, unless you don't want to make breaking changes or the current format is some kind of convention, I'd recommend renaming the files to e.g. `STRchive-disease-loci_v2.1.0_T2T-chm13_TRGT.bed`. To me that's more readable, understandable, and easily parsable with code, and will also group them together more nicely in alphabetical (most) file system views.

Reply:
Thanks for the reminder about file names @vincerubinetti 
What about something like this?

File names in github:
```			
STRchive-citations-test.json			
STRchive-citations.json				
STRchive-disease-loci.T2T-chm13.TRGT.bed
STRchive-disease-loci.T2T-chm13.bed
STRchive-disease-loci.hg19.TRGT.bed
STRchive-disease-loci.hg19.bed
STRchive-disease-loci.hg38.TRGT.bed
STRchive-disease-loci.hg38.bed
STRchive-loci.json
```

renaming:
```
            // add version to artifact filenames
            for (const file of globSync("**/STRchive-*.json"))
              renameSync(file, file.replace("STRchive", `STRchive_v${newVersion}`));
            for (const file of globSync("**/*STRchive-disease-loci*.bed"))
              renameSync(file, file.replace("STRchive", `STRchive_v${newVersion}`));
              
```
              
            
Which would produce:
`STRchive_v2.0.0-disease-loci.hg38.bed`

Reply:
That looks better to me, though I'd still replace the `_` with a `.`

Reply:
To be clear, I just mean that I'd be consistent with group vs. part separators. E.g.:

`STRchive-v2.0.0-disease-loci.hg38.bed` or `STRchive.v2.0.0.disease-loci.hg38.bed` or `STRchive_v2.0.0_disease-loci_hg38.bed`